r2i/tensorrt/meson.build

```diff
@@ -0,0 +1,33 @@
+# Define source code
+trt_sources = [
+  'engine.cc',
+  'frame.cc',
+  'frameworkfactory.cc',
+  'loader.cc',
+  'model.cc',
+  'parameters.cc',
+  'prediction.cc',
+]
+
+trt_headers = [
+  'engine.h',
+  'frame.h',
+  'frameworkfactory.h',
+  'loader.h',
+  'model.h',
+  'parameters.h',
+  'prediction.h',
+]
+
+# Build library
+trt_lib = static_library('trt',
+  trt_sources,
+  include_directories : [configinc, cuda_inc_dir],
```

Comment:
`cuda_inc_dir` should not be necessary if `lib_trt_dep` was configured appropriately.